server/db/models/users.js

```diff
@@ -0,0 +1,258 @@
+//define Users model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
+
+const { UUID, UUIDV4, STRING, BOOLEAN, INTEGER, DATEONLY } = Sequelize;
+
+const Users = db.define('users', {
+    userId: {
+        primaryKey: true,
+        type: UUID,
+        defaultValue: UUIDV4,
+    },
+    userType: {
+        type: STRING,
+        allowNull: false,
+        //may need to update this later depends on how a user chooses his type
+        validate: {
+            isIn: [
+                ['Guest', 'Existing customer', 'Admin']
+            ],
+        },
+    },
+    loggedIn: {
```

Comment:
default value of false

Reply:
Are you talking about the defaulstValue of userId? Could you clarify it?